iri/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "r2c2_iri"
```

Comment:
nit: it seems most crate are using `-` instead of `_` nowdays hence `r2c2-iri`

Reply:
looking at crates.io, it does not look like one is largely dominating the other...
I personally prefer the underscore (`_`) to keep the name of the crate consistent with how it is named in code.

But I won't die on that hill.

Reply:
Indeed, this seems to be unclear. My bad. I tend to prefer `-` for consistency with the keys in Cargo.toml I won't die on this hill either, let's choose something and stick with it.